test/test_utils.py

```diff
@@ -261,3 +262,36 @@ def get_formatted_current_timestamp(format: str = "%Y-%m-%d %H:%M:%S") -> str:
     # Format the timestamp as a human-readable string
     timestamp_str = current_time.strftime(format)
     return timestamp_str
+
+def list_items_in_folder(path):
+    try:
+        items = os.listdir(path)
+        items_with_mtime = [(item, os.path.getmtime(os.path.join(path, item))) for item in items]
+        items_with_mtime.sort(key=lambda x: x[1])
+        sorted_items = [item for item, mtime in items_with_mtime]
+        return sorted_items
+    except FileNotFoundError:
+        return f"The path {path} does not exist."
+    except NotADirectoryError:
+        return f"The path {path} is not a directory."
+    except PermissionError:
+        return f"Permission denied to access {path}."
+
+def compress_png(file_path, max_size_mb=20, reduce_factor=0.9):
```

Comment:
please document this method and some inline docs for the compression would be informative to others

Reply:
Added inline documentation.